08LessonEight/homework/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Delivery</title>
+    <link rel="stylesheet" href="css/exercise-06-style.css">
+</head>
+
+<body>
+  <div class="methods">
+    <div id="column">
+      <img src="img/truck.jpg" alt="truck">
+      <h4>SAFE & SECURE DELIVERY</h4>
+      <p>Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.</p>
+      <a class="info" target="_blank" href="#">READ MORE</a>
+    </div>
+    <div id="column">

```

Comment:
Remember, IDs are unique and cannot be assigned to more than one element, use` class="column"` instead